special-interest-groups/sig-community-infra/README.md

```diff
@@ -23,4 +23,4 @@ TBD
 
```

Comment:
The member list link is broken. (line 14)

Reply:
All the links are broken, maybe I should fix all the links in another pr.

Reply:
OK